website/docs/r/vapp_static_routing.html.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_static_routing"
+sidebar_current: "docs-vcd-resource-vapp-static-routing"
+description: |-
+  Provides a vCloud Director vApp static routing resource. This can be used to create, modify, and delete static routing rules.
+---
+
+# vcd\_vapp\_static\_routing
+
+Provides a vCloud Director vApp static routing resource. This can be used to create,
+modify, and delete static routing rules in a [vApp network](/docs/providers/vcd/r/vapp_network.html).
+
+~> **Note:** Resource used for vApp network, not vApp Org network.
+!> **Warning:** Using this resource overrides any existing static routing rules on vApp network. It's recommended to have only one resource per vApp. 
+
+## Example Usage
+
+```hcl
+resource "vcd_vapp" "web" {
+  name = "web"
+}
+​
+resource "vcd_vapp_network" "vapp-net" {
+  name               = "my-vapp-net"
+  vapp_name          = vcd_vapp.web.name
+  org_network_name   = "my-vdc-int-net"
+  gateway            = "192.168.2.1"
+  netmask            = "255.255.255.0"
+  dns1               = "192.168.2.1"
+
+  firewall_enabled = true
+  nat_enabled      = true
+​
+  static_ip_pool {
+    start_address = "192.168.2.51"
+    end_address   = "192.168.2.100"
+  }
+}
+
+resource "vcd_vapp_static_routing" "vapp1_static_routing" {
+  vapp_id    = vcd_vapp.web.id
+  network_id = vcd_vapp_network.vapp-net.id
+  enable     = false
```

Comment:
1. This parameter is undocumented below.
2. In all other resources we use the form `enabled`, when this single word is used.

Reply:
fixed

Reply:
Thanks. Why is it disabled in the example? If there's no special reason, I'd say having it enabled in the example is what I would expect when copy/pasting it to my config.

Reply:
good point, changed to true.

Reply:
Thanks. Thinking further, do we need to enable these for static routing to work?
```
resource "vcd_vapp_network" "vapp-net" {
<...>
  firewall_enabled = true
  nat_enabled      = true
```
If not, we should remove what's not needed to not confuse the users. Distill to the essence ;)

Reply:
removed